content/get-started/05_persisting_data.md

```diff
@@ -124,6 +153,24 @@ You can create the volume and start the container using the CLI or Docker Deskto
    $ docker run -dp 127.0.0.1:3000:3000 --mount type=volume,src=todo-db,target=/etc/todos getting-started
    ```
 
+{{< /tab >}}
+{{< tab name="Git Bash CLI" >}}
+
+1. Create a volume by using the `docker volume create` command.
+
+   ```console
+   $ docker volume create todo-db
+   ```
+
+2. Stop and remove the todo app container once again with `docker rm -f <id>`, as it is still running without using the persistent volume.
+
+3. Start the todo app container, but add the `--mount` option to specify a volume mount. Give the volume a name, and mount
+   it to `/etc/todos` in the container, which captures all files created at the path. In your Mac or Linux terminal, or in Windows Command Prompt or PowerShell, run the following command:
+
+   ```console
+   $ docker run -dp 127.0.0.1:3000:3000 --mount type=volume,src=todo-db,target=//etc/todos getting-started
```

Comment:
can we avoid the double slash by quoting `--mount` option?

Reply:
I tried both of the following and they didn't work:
`docker run -dp 127.0.0.1:3000:3000 --mount "type=volume,src=todo-db,target=/etc/todos" getting-started`

`docker run -dp 127.0.0.1:3000:3000 --mount type=volume,src=todo-db,target="/etc/todos" getting-started`

Reply:
Based on the other comment, removed the CLI tabs here. Used a note to single out Git Bash in order to make it less verbose by not naming all the CLIs when only Git Bash is the odd ball.